doc/testing_samples/sidewalk_application_samples/template_subghz.rst

```diff
@@ -230,25 +234,25 @@ To select the build type when building the application from command line, specif
 
    .. code-block:: console
 
-      $ west build -b nrf52840dk_nrf52840
+      $ west build -b build_target
```

Comment:
```suggestion
      .. parsed-literal::
   :class: highlight

   west build -b *build_target*

For example:

.. code-block:: console

   west build -b nrf52840dk_nrf52840
```
I saw this in matter readme files, I think it's good idea to write _build_target_ with different font type, as it isn't a supported board name in west

Reply:
good point - will update it and add an example for each first occurrence of a build target per sample :)